xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -31,15 +31,30 @@
   
```

Comment:
not related to your change but remove self-link from the remarks paragraph above?